src/handlers/api/openapi.json

```diff
@@ -0,0 +1,312 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Lockbot",
+    "description": "Coordinate use of your team's shared resources, in Slack 🤝 ... or via HTTP API",
+    "contact": {
+      "name": "Lockbot.app",
+      "url": "https://lockbot.app/"
+    },
+    "version": "1.0.0"
+  },
+  "externalDocs": {
+    "description": "GitHub Repository",
+    "url": "https://github.com/connorads/lockbot"
+  },
+  "servers": [
+    {
+      "url": "https://2p30aws1sc.execute-api.eu-west-1.amazonaws.com/dev/api/",
+      "description": "dev"
+    },
+    {
+      "url": "https://xjj5kgewr8.execute-api.eu-west-1.amazonaws.com/prod/api/",
+      "description": "prod"
+    }
+  ],
```

Comment:
Remove this if we can get it hosted in each environment probs not needed and can be based on current `location`/`host`